Sender_BMS/GenerateParemeters.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+
+
+namespace Sender_BMS
+{
+    public class GenerateParemeters: IInputStreamer
```

Comment:
This is a class without data-members. Why is it needed to make a `new` of this everytime we need to use it?